pages/index.tsx

```diff
@@ -1,15 +1,26 @@
 import Link from "next/link";
 import Typography from "@material-ui/core/Typography";
-import { MainChartListContainer } from "../pods/main-chart-list";
+import { MainChartListContainer, getAllCharts } from "../pods/main-chart-list";
 
-export default function Index() {
+const Index = props => {
   return (
     <div>
       <Typography variant="h2" gutterBottom>
         Página principal gráficas
       </Typography>
 
-      <MainChartListContainer />
+      <MainChartListContainer charts={props.charts} />
     </div>
   );
 }
+
+Index.getInitialProps = async () => {
```

Comment:
Same thing here we have to type this I guess

Reply:
I would need some help over here.
I wrote this like this because of the [documentation](https://nextjs.org/learn/basics/fetching-data-for-pages), it says that this is how you fetch data from an API.

But another way its to remove this, and get the data using a react hook on the `main-chart-list.container.tsx`.

What do you think?